File: firstcoursework/app/src/main/java/com/global/firstcoursework/MainActivity.java
package com.global.firstcoursework;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.TextView;


public class MainActivity extends AppCompatActivity{

    TextView txt1;
    TextView txt2;
    TextView txt3;
    TextView txt4;
    TextView txt5;
    TextView txt6;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        txt1 = (TextView) findViewById(R.id.txt1);
        txt2 = (TextView) findViewById(R.id.txt2);
        txt3 = (TextView) findViewById(R.id.txt3);
        txt4 = (TextView) findViewById(R.id.txt4);
        txt5 = (TextView) findViewById(R.id.txt5);
        txt6 = (TextView) findViewById(R.id.txt6);

        txt1.setText("Computer and network security");
        txt2.setText("Data and Communications networks");
        txt3.setText("Entrepreneurship");
        txt4.setText("Mobile programming");
        txt5.setText("Artificial intelligence");
        txt6.setText("Automata, Complexity and Computability");
    }
}

File: firstassignment2/app/src/main/java/com/global/firstassignment2/MainActivity.kt
package com.global.firstassignment2

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.TextView


class MainActivity : AppCompatActivity(){
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val text1: TextView = findViewById(R.id.txt1) as TextView
        val text2: TextView = findViewById(R.id.txt2) as TextView
        val text3: TextView = findViewById(R.id.txt3) as TextView
        val text4: TextView = findViewById(R.id.txt4) as TextView
        val text5: TextView = findViewById(R.id.txt5) as TextView
        val text6: TextView = findViewById(R.id.txt6) as TextView

        text1.setText("Computer and Network  Security")
        text2.setText("Data communications and networks")
        text3.setText("Entrepneurship")
        text4.setText("Mobile Programming")
        text5.setText("Artificial Intelligence")
        text6.setText("Automata,Complexity and Computability")

    }
}